### DISCUSSION TITLE
Possible Missing Buildings in the "What kind of building is this?" Quest

### DISCUSSION THREAD

**Discussion by bpezer** [streetcomplete/StreetComplete#5364] [Feedback]
Hello everybody. I started using StreetComplete again recently after a couple of years of pause and have noticed a couple of possible missing buildings. Both of them aren't particularly common, so I would like to consult with others and figure out if I'm supposed to tag them differently (and if so, how), or if these two building categories should be added to the quest.

- [Lighthouse](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dlighthouse)
- [Loculus](https://en.wikipedia.org/wiki/Loculus_(architecture))

Regarding the Lighthouse, I can imagine it could be tagged as a **Service building** (already present in the app), but wouldn't it be better to be more precise?

As for the loculus (of which there are many in the area where I'm living), the buildings in the religious category feel too vague. They are definitely not church buildings, nor shrines. They could maybe be considered as mausoleums (but this category is also not present in the app).

P.S. If the decision were made to add these buildings to the quest, I would volunteer to do so.
P.P.S. I'm sure I noticed a couple of other building categories missing, but I can't recall at the moment. If they come to mind I'll mention them in the discussion.

**Comment by mnalis** [comment]
**TL;DR** - in StreetComplete leave a Note for them (preferably with a picture), and later you (or someone more advanced in OSM) can add in all missing details, and not _**just**_ a (less useful) architectural type of some building. Or try [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door) app too.

Want all the gory details? Read on.

---



> [Lighthouse](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dlighthouse)
    [Loculus](https://en.wikipedia.org/wiki/Loculus_(architecture))
> Both of them aren't particularly common

Yeah, there ain't enough space (nor patience in users) to list [thousands of possible building types](https://taginfo.openstreetmap.org/keys/building#values), so StreetComplete offers only the most popular ones, giving you `Other answers` / `Can't say` / `Leave note` as a way to leave a Note, so it can be added to the map later (by you in some more advanced editor, or by anybody else).

What is important to remember is that this quest only **tags only architectural type of the building** (as noted in e.g.  https://github.com/streetcomplete/StreetComplete/issues/5112#issuecomment-1636777699), **and _not_ its use**, so even if there was specific picture of lighthouse in SC to tag [building=lighthouse](https://wiki.openstreetmap.org/wiki/Tag:building%3Dlighthouse),  you'd still want to leave a note in case lighthouse  _itself_ (i.e. [man_made=lighthouse](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dlighthouse)) was not mapped (otherwise, people searching for lighthouses won't find them on the map).


StreetComplete does offer (in somewhat misnamed `Shops overlay`) to add actual features (like churches) if they are missing; but it currently only allows for some of them (although there are plans to support more features like lighthouses in different overlay in the future, see https://github.com/streetcomplete/StreetComplete/issues/4912 for details).

For advanced users, [SCEE](https://wiki.openstreetmap.org/wiki/SCEE) (_"expert edition"_ fork of StreetComplete) does allow for more opportunities of direct editing on the field instead of leaving notes. Another popular alternative (whose workflow is better suited for this task IMHO than StreetComplete and its forks) to see what exactly is mapped and what is missing, and to add/update  Points Of Interest is [Every Door](https://wiki.openstreetmap.org/wiki/Every_Door).


> As for the loculus (of which there are many in the area where I'm living), the buildings in the religious category feel too vague. They are definitely not church buildings, nor shrines. They could maybe be considered as mausoleums (but this category is also not present in the app).

Note that you can select category ***itself*** for less-precise answers in cases where more precise answer is missing from the list. E.g. if `church` nor `shrine` do not fit the bill, you can select `Religious building` category as select that, and SC will tag more general [building=religious](https://wiki.openstreetmap.org/wiki/Tag:building=religious) on it.

But see the warning above - if the loculus itself isn't mapped (i.e. there is no [amenity=grave_yard](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dgrave_yard) or [historic=tomb](https://wiki.openstreetmap.org/wiki/Tag:historic%3Dtomb) + [tomb=*](https://wiki.openstreetmap.org/wiki/Key:tomb) or what have you), just marking specific architectural type of the building is not very useful.

In case you're now wondering  _"why is the building type even asked then if it is not very useful?"_; the short answer is to to save mapper time -- i.e. to skip asking many other quests which are not relevant (e.g. it does not make sense to ask what is the housenumber of a `shed`, what is street of a `shed`, what is number of floors of `shed` etc).

**Reply by bpezer** [reply]
Everything is clear now, thanks a lot for the detailed explanation.

**Reply by mnalis** [reply]
Updated [StreetComplete FAQ: "Building Type Quest - Why don't I see the right types?"](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Building_Type_Quest_-_Why_don%27t_I_see_the_right_types%3F)

**Reply by FloEdelmann** [reply]
Really well written FAQ entry!
